packages/openapi-generator/package.json

```diff
@@ -38,8 +38,9 @@
     ]
   },
   "scripts": {
-    "compile": "yarn tsc -b && oclif-dev readme && cp -r -f src/templates dist",
+    "compile": "yarn tsc -b && cp -r -f src/templates dist",
     "prepare": "yarn compile",
+    "readme": "oclif-dev readme",
```

Comment:
This should be run automatically somewhere, so that we can't forget to update it.

Reply:
yes. I will include it in the postinstall step, ok?

Reply:
I think that should be ok